include/library/spdm_requester_lib.h

```diff
@@ -370,6 +370,40 @@ libspdm_return_t libspdm_get_measurement_extension_log(void *spdm_context,
                                                        void *measure_exten_log);
 #endif /* LIBSPDM_ENABLE_CAPABILITY_MEL_CAP */
 
+#if LIBSPDM_ENABLE_CAPABILITY_GET_KEY_PAIR_INFO_CAP
+/**
+ * This function sends GET_KEY_PAIR_INFO to get key pair info from device.
+ *
+ * @param  spdm_context               A pointer to the SPDM context.
+ * @param  session_id                 Indicates if it is a secured message protected via SPDM session.
+ *                                    If session_id is NULL, it is a normal message.
+ *                                    If session_id is not NULL, it is a secured message.
+ * @param  key_pair_id                Indicate which key pair ID's information to retrieve.
+ *
+ * @param  total_key_pairs            Indicate the total number of key pairs on the responder.
+ * @param  capabilities               Indicate the capabilities of the requested key pairs.
+ * @param  key_usage_capabilities     Indicate the key usages the responder allows.
+ * @param  current_key_usage          Indicate the currently configured key usage for the requested key pairs ID.
+ * @param  asym_algo_capabilities     Indicate the asymmetric algorithms the Responder supports for this key pair ID.
+ * @param  current_asym_algo          Indicate the currently configured asymmetric algorithm for this key pair ID..
+ * @param  public_key_info_len        On input, indicate the size in bytes of the destination buffer to store.
+ *                                    On output, indicate the size in bytes of the public_key_info.
+ * @param  assoc_cert_slot_mask       This field is a bit mask representing the currently associated certificate slots.
+ * @param  public_key_info            A pointer to a destination buffer to store the public_key_info.
+ **/
+libspdm_return_t libspdm_get_key_pair_info(void *spdm_context, const uint32_t *session_id,
+                                           uint8_t key_pair_id, uint8_t *total_key_pairs,
+                                           uint16_t *capabilities,
+                                           uint16_t *key_usage_capabilities,
+                                           uint16_t *current_key_usage,
+                                           uint32_t *asym_algo_capabilities,
+                                           uint32_t *current_asym_algo,
+                                           uint16_t *public_key_info_len,
```

Comment:
Swap position of `public_key_info_len` and `assoc_cert_slot_mask`. `public_key_info_len` and `public_key_info` should be adjacent to one another.